teps/0015-pause-pipeline.md

```diff
@@ -0,0 +1,105 @@
+---
+title: pause-pipeline
+authors:
+  - "@jbarrick-mesosphere"
+creation-date: 2020-09-10
+last-updated: 2020-09-10
+status: draft
+---
+
+# TEP-0015: Support Paused PipelineRuns and TaskRuns
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Requirements](#requirements)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [Platform Implementor](#platform-implementor)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+This TEP proposes a method for pausing PipelineRuns and TaskRuns that are then
+ignored by the Tekton reconciler. These paused resources can be used by platform
+implementers to control PipelineRun and TaskRun resources, for example, in cases
+where the cluster is under heavy load and not ready to process a PipelineRun.
+
+PipelineRuns and TaskRuns can be created paused so that they are not started at
+all or, in the case of a PipelineRun, the pause setting can be used to prevent
+further TaskRuns from being created. As it is not possible to "pause" a
```

Comment:
I think we only need `pause` in `PipelineRun` but not `TaskRun` since `runAfter` only available in `PipelineRun`. Once a `PipelineRun` is paused, after the completion of the current pipeline, the next pipeline that runs after
the previous one will be paused, but other pipelines won't be affected.